packages/manager/modules/pci/src/projects/new/config/controller.js

```diff
@@ -18,26 +20,64 @@ export default class PciProjectNewConfigCtrl {
     }).join(', ');
   }
 
-  /* -----  End of Helpers  ------ */
-
-  /* =============================
-  =            Events            =
-  ============================== */
-
-  onProjectConfigFormSubmit() {
-    let setDescrPromise = Promise.resolve(true);
-
+  setCartProjectItem() {
     if (
       this.model.description &&
       !this.cart.projectItem.descriptionConfiguration
     ) {
-      setDescrPromise = this.pciProjectNew.setCartProjectItemDescription(
+      return this.pciProjectNew.setCartProjectItemDescription(
```

Comment:
No more redirection to payment ?

Reply:
yes i refacto a bit the controller.
so the `goToPayment` function  it goes here:
https://github.com/ovh/manager/blob/b4f6d8f00350e14eaecc62c989f27e59e17cc807/packages/manager/modules/pci/src/projects/new/config/controller.js#L80